generate.py

```diff
@@ -57,29 +59,47 @@ def main():
         residual_channels=wavenet_params['residual_channels'],
         dilation_channels=wavenet_params['dilation_channels'],
         quantization_channels=wavenet_params['quantization_channels'],
-        use_biases=wavenet_params['use_biases'])
+        use_biases=wavenet_params['use_biases'],
+        fast_generation=args.fast_generation)
 
     samples = tf.placeholder(tf.int32)
 
     next_sample = net.predict_proba(samples)
-
-    saver = tf.train.Saver()
+    
+    if args.fast_generation:
+        sess.run(tf.initialize_all_variables())
```

Comment:
`Saver` initializes the variables which was not restored, so this line is not necessary.
(@ibab let me know, though 😃: https://github.com/ibab/tensorflow-wavenet/issues/13#issuecomment-247645912)